_posts/2021-02-10-hindsight-on-2020.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+author: HeroicKatora
+title: "The last and next year of image-rs"
+co_authors: [Shnatsel]
+---
+
+The last year was mainly one of stability. Working up to `0.23` we had fixed
+many outstanding stability risks and it continues to be one of the most
+long-serving major versions in a while. In this post you find an overview on
+the massive progress made with it, and outstanding issues that you could
+contribute towards in 2021.
+
+We also have a logo now:
+
+<div style="background: center/contain no-repeat url('/assets/image-rs-logo.png'); width:100%; height:10em;"></div>
+
+## The gif crate now 2x faster
+
+The `gif` crate joins `png` in being on par with the relevant C implementation
+in terms of performance!
+
+These gains are thanks to a new LZW encoding/decoding library, [called
+`weezl`][weezl]. It significantly improves on lzw crate in performance while
+staying 100% safe, and also had an optional `no_std` mode. There is integration
+with both `std` and `async` IO-interfaces, behind separate features. `weezl`,
+written by the author of this post, is also maintained under the image-rs
+organization that hosts a lot of format-specific image encoding/decoding
+crates.
+
+Since most of the time in decoding GIF files is spent in LZW decoding, the
+gains from LZW have naturally translated to gif crate. TIFF files encoded with
+LZW compression also benefit from this change. The latest versions of gif, tiff
+and image all take advantage of this new library.
+
+[weezl]: https://crates.io/crates/weezl
+
+## Developments in 2020
+
+Part of the stability work was the ability to upgrade decoder dependencies
+behind the scenes. This allowed you to silently benefit from the improved `gif`
+decoder as detailed above and other similar fixes across other decoders and it
+will also enable similar updates for `png` soon.
+
+This is directly reflected in dependencies and downloads. While previously
+there was a long tail, the dependencies and downloads are now [dominantly for
+recent versions](https://lib.rs/crates/image/rev) and thus quicker, more
+secure, and more correct. (If you're still using `0.18` or `0.22`, you should
+_really_ consider updating). We've also more than doubled our total downloads
+over the last year!
+
+All of this didn't stop progress. Take the `png` and `tiff` decoders that have
+seen a significant increase in coverage of features. Add the fact that `png`
```

Comment:
To make the achievements even more impressive, spread them across categories instead of a single paragraph with everything lumped together. Rough outline to provide an example:

**Performance**

gif and png are on par with C, improvements to jpeg but it's hard to compete with SIMD assembly; huffman decoding is slowing things down, it's over here, please help us optimize it.

**Features**

<list and description of the newly added features goes here>

**Stability**

<the image stable API goes here>

**Compatibility**

<bragging about running hundreds of thousands of images through each decoder goes here>